src/src/DataStructs/ESPEasyDefaults.h

```diff
@@ -89,6 +89,10 @@
 #ifndef DEFAULT_CONTROLLER
 #define DEFAULT_CONTROLLER   false              // true or false enabled or disabled, set 1st controller defaults
 #endif
+
+#ifndef DEFAULT_CONTROLLER_ENABLED
+#define DEFAULT_CONTROLLER_ENABLED   false     // Enable default controller by default
```

Comment:
Fine to have it in the sample, but not as default for anyone.

Reply:
This is the reason why the value is by default false and offers the user the possibility to change it via Custom.h.
When rereading, it is true that the commentary is not very explicit

Reply:
Ah, now I get it.
I will think about the comments, since it is a bit confusing right now.